tests/test_pandas_api.py

```diff
@@ -1775,6 +1775,27 @@ def test_pandas_abs(kx, q):
         tab.abs()
 
 
+def test_pandas_round(kx, q):
+    q_tab = q('([]c1:1.0 .1 .01 .001 .0001 .00001;'
+              'c2:1.0 1.888 1.02 1.999 1.0002 1.00002;'
+              'c3:til 6;'
+              'c4:0 0.11 0.22 0.33 0.44 0.55e;'
+              'c5:`a`b`c`d`e`f)')
+    pd_tab = q_tab.pd()
+    round_dict = {'c1': 0, 'c2': 2}
+
+    assert all(pd_tab.round() == q_tab.round().pd())

```

Comment:
Consider using another way of testing because with that method, the test 
`assert all(pd_tab.round() == pd_tab.round(3))` pass, when it should not.
You can use methods like: `pd.testing.assert_frame_equal(a,b)`

Reply:
Done